readme.md

```diff
@@ -2,6 +2,8 @@
 
 The most trusted platform to store digital assets on Ethereum
 
+[![Coverage Status](https://coveralls.io/repos/github/gnosis/safe-react/badge.svg?branch=development)](https://coveralls.io/github/gnosis/safe-react?branch=development)
```

Comment:
shouldn't it be branch=master?

Reply:
Yes I suppose but we couldn't check if the badge it's working before merging it to master, I would suggest leaving as it is just to make it work, and then if everything is ok I open another pr to update that value to master